ocs_ci/helpers/helpers.py

```diff
@@ -2162,9 +2162,15 @@ def create_multiple_pvc_parallel(sc_obj, namespace, number_of_pvc, size, access_
     # Check for all the pvcs in Bound state
     with ThreadPoolExecutor() as executor:
         for objs in pvc_objs_list:
-            obj_status_list.append(
-                executor.submit(wait_for_resource_state, objs, "Bound", 90)
-            )
+            if type(objs) is list:
+                for obj in objs:
+                    obj_status_list.append(
+                        executor.submit(wait_for_resource_state, obj, "Bound", 90)
```

Comment:
would status always be `Bound` for any instance?

Reply:
Why any instance. This part is inside a function to create PVC. I don't see any other instance getting here unless you are really really talented.